drivers/SmartThings/zwave-switch/src/zwave-dual-switch/init.lua

```diff
@@ -11,29 +11,29 @@
 -- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
-
+local st_device = require "st.device"
 local capabilities = require "st.capabilities"
---- @type st.utils
-local utils = require "st.utils"
---- @type st.zwave.constants
-local constants = require "st.zwave.constants"
+--- @type st.zwave.defaults.switch
+local switch_defaults = require "st.zwave.defaults.switch"
 --- @type st.zwave.CommandClass
 local cc = require "st.zwave.CommandClass"
 --- @type st.zwave.CommandClass.Basic
-local Basic = (require "st.zwave.CommandClass.Basic")({ version = 1 })
+local Basic = (require "st.zwave.CommandClass.Basic")({ version = 1, strict=true })
 --- @type st.zwave.CommandClass.SwitchBinary
-local SwitchBinary = (require "st.zwave.CommandClass.SwitchBinary")({ version = 2 })
+local SwitchBinary = (require "st.zwave.CommandClass.SwitchBinary")({ version = 2, strict = true })
+--- @type st.zwave.CommandClass.Meter
+local Meter = (require "st.zwave.CommandClass.Meter")({ version = 3 })
 
 local ZWAVE_DUAL_SWITCH_FINGERPRINTS = {
-  {mfr = 0x0086, prod = 0x0103, model = 0x008C}, -- Aeotec Switch 1
-  {mfr = 0x0086, prod = 0x0003, model = 0x008C}, -- Aeotec Switch 1
-  {mfr = 0x0258, prod = 0x0003, model = 0x008B}, -- NEO Coolcam Switch 1
-  {mfr = 0x0258, prod = 0x0003, model = 0x108B}, -- NEO Coolcam Switch 1
-  {mfr = 0x0312, prod = 0xC000, model = 0xC004}, -- EVA Switch 1
-  {mfr = 0x0312, prod = 0xFF00, model = 0xFF05}, -- Minoston Switch 1
-  {mfr = 0x0312, prod = 0xC000, model = 0xC007}, -- Evalogik Switch 1
-  {mfr = 0x010F, prod = 0x1B01, model = 0x1000}, -- Fibaro Walli Double Switch
-  {mfr = 0x027A, prod = 0xA000, model = 0xA003} -- Zooz Double Plug
+  { mfr = 0x0086, prod = 0x0103, model = 0x008C }, -- Aeotec Switch 1
+  { mfr = 0x0086, prod = 0x0003, model = 0x008C }, -- Aeotec Switch 1
+  { mfr = 0x0258, prod = 0x0003, model = 0x008B }, -- NEO Coolcam Switch 1
+  { mfr = 0x0258, prod = 0x0003, model = 0x108B }, -- NEO Coolcam Switch 1
+  { mfr = 0x0312, prod = 0xC000, model = 0xC004 }, -- EVA Switch 1
+  { mfr = 0x0312, prod = 0xFF00, model = 0xFF05 }, -- Minoston Switch 1
+  { mfr = 0x0312, prod = 0xC000, model = 0xC007 }, -- Evalogik Switch 1
+  { mfr = 0x010F, prod = 0x1B01, model = 0x1000 }, -- Fibaro Walli Double Switch
+  { mfr = 0x027A, prod = 0xA000, model = 0xA003 } -- Zooz Double Plug
```

Comment:
this device should be handled by another driver (according to DTH). https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/78747fa5a3b42a0c801f1095f8f7521da968ba42/devicetypes/smartthings/zwave-multi-metering-switch.src/zwave-multi-metering-switch.groovy#L32